microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+    ![The Lighthouse tool report showing a better score after the changes](./images/lighthouse-report-without-map.png)
 
-Optimization is when you run an image through a special program that reduces the size of the image file.  For more tips, see [Essential Image Optimization](https://images.guide).
+    Your **Lighthouse** score is now at 31, which is an improvement from before, but there are more things to optimize.
 
-### Eliminate render-blocking resources
+#### Eliminate render-blocking resources
 
-Your latest report says that eliminating render-blocking resources is now the biggest opportunity.
+The next opportunity displayed in the **Opportunities** section of the **Lighthouse** tool is related to eliminating render-blocking resources.
 
-A render-blocking resource is an external JavaScript or CSS file that the browser must download, parse, and run before it displays the page.  The goal is to only run the core CSS and JavaScript code that is required to display the page properly.
+A render-blocking resource is an external JavaScript or CSS file that the browser must download, parse, and run before it displays the page. The more render-blocking resources need to be processed when a webpage loads, the longer the webpage needs to start appearing in the browser. Try to only run the core CSS and JavaScript code that's required to display the initial state of the page properly.
 
 The first task, then, is to find code that you don't need to run on page load.
 
-1. Click **Eliminate render-blocking resources** to display the resources that are blocking: `lodash.js` and `jquery.js`.
+1. Click **Eliminate render-blocking resources** to display the resources that are blocking:
 
-   ![More information about the Eliminate render-blocking resources opportunity](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded.msft.png)
+   ![More information about the Eliminate render-blocking resources opportunity](./images/render-blocking-css.png)
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the Command Menu, start typing `Coverage`, and then select **Show Coverage**.
+    **Lighthouse** displays a list of the stylesheets that the demo webpage uses, such as: `base.css`, `home.css`, `map.css`, and others.
 
-   ![Open the Command Menu from the Audits panel](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded-command-coverage.msft.png)
+1. Open the **Coverage** tool again: press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS), type `Coverage`, and then select **Show Coverage**.
 
-   ![The Coverage tool](../media/speed-glitch-tony-remix-updated-audits-performance-oppportunities-expanded-drawer-coverage.msft.png)
+1. Click **Start instrumenting coverage and refresh the page** (![Refresh icon](../media/reload-icon.msft.png)) to display the coverage report, and then type `css` in the **URL filter** field to only display the CSS files:
```

Comment:
I would rather do this in a repro-wide image cleanup task (possible after Focus Mode) rather than here. If this image is used in many other places (which it probably is), this will increase this PR's size substantially for unrelated things.